source/clear-linux/concepts/security.rst

```diff
@@ -161,7 +161,7 @@ good security hygiene. Below are some examples:
 * Full disk encryption using `Linux Unified Key Setup`_ (LUKS)  is available 
   during installation.
 
-* Use the PAM cracklib module to harden user login and password security: 
+* Use of the PAM cracklib module to harden user login and password security: 
```

Comment:
Line 164: why is the word "of" added?

Reply:
"Use the PAM cracklib..." sounds like a command/action to the reader when this is meant to more explanatory about *how* this is achieved. (i.e. Clear Linux makes use of the the PAM cracklib).

Let me know what you think is the best way to rephrase if needed.

Reply:
Got it. I think you already did a pretty good job: Clear Linux uses the PAM cracklib ...